docs/deployment/uwsgi_cookiecutter_1_nginx.rst

```diff
@@ -38,7 +38,7 @@ With that said, let's begin.
 
     .. code-block:: bash
 
-        $ cd
+        $ cd ~
```

Comment:
I often use the plain `cd` to get to my home directory. Perhaps this is ambiguous. Changing it to `cd ~`.